doc/appendix-de-kassensichv/cash-register-integration/receipt-sequences-creation.md

```diff
@@ -0,0 +1,174 @@
+---
+slug: /poscreators/middleware-doc/germany/receipt-sequences-creation
+title: Receipt sequences creation
+---
+
+# Receipt sequences creation
+
+In the chapter [Single receipt creation](https://docs.fiskaltrust.cloud/docs/poscreators/middleware-doc/germany/single-receipt-creation), the creation of single receipts using either implicit and/or explicit flow has been described.
```

Comment:
link does not work for me

Reply:
it will work once the PR is merged (hopefully...)